packages/server/tsconfig.json

```diff
@@ -9,16 +9,33 @@
       "parabol-client/*": ["client/*"],
       "~/*": ["client/*"]
     },
-    "lib": ["esnext", "dom"],
-    "types": ["node", "jest", "jest-extended"]
+    "lib": [
+      "esnext",
+      "dom"
+    ],
+    "types": [
+      "node",
+      "jest",
+      "jest-extended"
+    ]
   },
-  "exclude": ["**/node_modules", "types/githubTypes.ts"],
+
+  "exclude": [
+    "**/node_modules",
+    "types/githubTypes.ts",
+    "postgres/migrationTemplate.ts",
+    "graphql/intranetSchema/sdl/resolverTypes.ts"
+  ],
   "files": [
     "types/webpackEnv.ts",
     "types/modules.d.ts",
     "../client/modules/email/components/SummaryEmail/MeetingSummaryEmail/MeetingSummaryEmail.tsx",
     "graphql/queries/tasks.ts",
     "graphql/queries/archivedTasks.ts",
+    "graphql/types/AzureDevOpsWorkItem.ts",
+    "graphql/types/AzureDevOpsRemoteProject.ts",
+    "graphql/types/AzureDevOpsIntegration.ts",
+    "dataloader/azureDevOpsLoaders.ts",
```

Comment:
-1 adding these files manually should not be necessary since it is included by others already